apps/condo/domains/ticket/access/TicketFilterTemplate.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Generated by `createschema ticket.TicketFilterTemplate 'name:Text; employee:Relationship:OrganizationEmployee:CASCADE; filters:Json'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { getByCondition, getById } = require('@core/keystone/schema')
+
+async function canReadTicketFilterTemplates ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return {}
+
+    return {
+        employee: { user: { id: user.id } },
+    }
+}
+
+async function canManageTicketFilterTemplates ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    if (operation === 'create') {
+        const employeeForUser = await getByCondition('OrganizationEmployee', {
+            id: originalInput.employee.connect.id,
+            user: { id: user.id },
+            deletedAt: null,
+            isBlocked: false,
+        })
+
+        if (!employeeForUser || employeeForUser.isBlocked) {
```

Comment:
`employeeForUser` will never have `isBlocked` field with `true` value here, because it's already filtered out by query above.
```suggestion
        if (!employeeForUser) {
```